src/hostedPages/activities/form/TraditionalForm.tsx

```diff
@@ -16,6 +16,7 @@ export const TraditionalForm = ({
   storedAnswers,
   onAnswersChange,
   autosaveAnswers = true,
+  trademark,
```

Comment:
Isn't it easier to just access trademark as a prop of `form` instead of adding again another property to the component?

```typescript
          {form?.trademark && (
            <div className={`${classes.trademark} ${classes.conversational}`}>
              {form.trademark}
            </div>
          )}
```

Reply:
Yes, moved too quickly and missed that trademark is part of the form already!